Cartfile

```diff
@@ -0,0 +1 @@
+github "joshaber/SwiftBox" "c11e489"
```

Comment:
Ah, yeah, SwiftBox needs a few other changes so I can take care of that later :+1: